docs/user_guide/5_changelog.md

```diff
@@ -2,6 +2,31 @@
 
 
 ## ROS Noetic
+
+### Version 1.0.18 (current noetic-release)
+* Fixing demo behaviour when tactile sensors are installed (1)
+* Fixing demo behaviour when tactile sensors are installed (2)
+
+### Version 1.0.17
+* Update repository.rosinstall
+* Fix handling of active rosbags
+* Remove default hand serial parameter
+* Remove sr_ethercat_config
+* Fixing bug config file pid parameters being erased when saving selected
+* Rounding up values for joint slider
+
+### Version 1.0.16
+* Fixed linter errors (1)
```

Comment:
If you have multiple occurrences of "Fixed linter errors" you can just squash them into one.

Reply:
Done